config-controller/pkg/client/client.go

```diff
@@ -0,0 +1,303 @@
+package client
+
+//go:generate mockgen-wrapper CachedPolicyClient,PolicyClient
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/go-logr/logr"
+	"github.com/pkg/errors"
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	storage "github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/clientconn"
+	"github.com/stackrox/rox/pkg/env"
+	"github.com/stackrox/rox/pkg/mtls"
+	"github.com/stackrox/rox/pkg/retry"
+	"github.com/stackrox/rox/pkg/size"
+	"google.golang.org/grpc"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+var centralHostPort = fmt.Sprintf("central.%s.svc:443", env.Namespace.Setting())
```

Comment:
There could be an env var to read for this instead of hard coding this. Wary of the hard coding.

Reply:
So thinking out loud here, should this be an env var setting in `pkg/env`?  How would the default value be populated as above? could it "depend" on another env var value?